packages/pangea-node-sdk/package.json

```diff
@@ -15,7 +15,7 @@
   "license": "MIT",
   "dependencies": {
     "crypto-js": "^4.1.1",
-    "got": "11.8.5",
+    "got": "12.6.1",
```

Comment:
Upgrading `got` to 12 will break support for common js style code, version 12 is esm only.

Reply:
Oh! Okey. I've just checked that updating to 11.8.6 should be enough to avoid vulnerability reported. I'll make a new PR with that change